anndata/readwrite/utils.py

```diff
@@ -1,10 +1,38 @@
-from functools import wraps
-from .. import h5py as patched_h5py
+from functools import wraps, singledispatch
+from ..core.sparsedataset import SparseDataset
 
 # -------------------------------------------------------------------------------
 # Type conversion
 # -------------------------------------------------------------------------------
 
+# Could be numba'd if it returned tuples instead of slices
+def idx_chunks_along_axis(shape: tuple, axis: int, chunk_size: int):
```

Comment:
Where is this used?

Reply:
It's used in all the dense <-> sparse io functions. Github hides those diffs by default due to length (probably since a lot of signatures got modified).

Reply:
Ah, I missed that hidden diff as it came after the diffs hidden for deleted files.